src/main/java/com/firebolt/jdbc/resultset/FireboltResultSet.java

```diff
@@ -106,15 +108,16 @@ private FireboltResultSet() {
 		columns = new ArrayList<>();
 		statement = null;
 		maxRows = 0;
+		maxFieldSize = 0;
```

Comment:
nit: add a comment saying that 0 is unlimited.

Reply:
added comment